src/core/election.rs

```diff
@@ -0,0 +1,58 @@
+use std::collections::HashMap;
+use std::hash::Hash;
+
+pub struct Election<T> {
+    pub electors_votes: HashMap<String, Option<T>>,
+}
+
+impl<T> Election<T> {
```

Comment:
Could we add documentation strings ([rustdoc?](https://doc.rust-lang.org/rustdoc/what-is-rustdoc.html)) for our classes and methods? These are starting to become non trivial and docs should be useful.

Reply:
Yes, I could and I will. Good remark jimbo

Reply:
this is done